test-packages/test-services/srv/v4/test-service/TestComplexType.ts

```diff
@@ -102,7 +102,7 @@ export interface TestComplexType {
 /**
  * @deprecated Since v1.6.0. Use [[TestComplexType.build]] instead.
  */
-export function createTestComplexType_1(json: any): TestComplexType {
+export function createTestComplexType(json: any): TestComplexType {
```

Comment:
The reason here is, that in the v2 example we have a test to check that names of function imports do not clash with names of the builders of complex types. Initially we just copied the v2 service. I now removed the case that tests the name formatting from v4 because I thought it would be the same. Maybe I am wrong thought, what do you think?